msbuild/Xamarin.MacDev.Tasks/Tasks/AOTCompileTaskBase.cs

```diff
@@ -34,13 +34,13 @@ public abstract class AOTCompileTaskBase : XamarinTask {
 		[Required]
 		public string SdkDevPath { get; set; } = string.Empty;
 
-#region Output
+		#region Output
```

Comment:
Nit: this one IIRC does go to the very left

Reply:
There's no editorconfig option to choose to not indent #regions 😒 : https://github.com/dotnet/roslyn/issues/37575